tests/ci/cdk/cdk.context.json

```diff
@@ -2,7 +2,7 @@
   "acknowledged-issue-numbers": [
     19836
   ],
-  "availability-zones:account=431229050501:region=us-west-2": [
+  "availability-zones:account=620771051181:region=us-west-2": [
```

Comment:
This is just metadata that gets updated by CDK regardless, I just didn't want my personal account in here :)